hermes/file_gen.go

```diff
@@ -0,0 +1,38 @@
+package hermes
+
+import (
+	"fmt"
+	"strconv"
+	"crypto/sha1"
+)
+type HermesFile struct {
+	name string
+	contents string
+}
+func (file *HermesFile) GenerateFileName(file_ID int, file_checksum string) {
+	id_string :=""
+	if file_ID < 10 {
+		id_string = "0" + strconv.Itoa(file_ID)
+	} else {
+		id_string = strconv.Itoa(file_ID)
+	}
+	file_name := "Hermes_"+id_string+file_checksum
+	file.name = file_name
+}
+func (file *HermesFile) GenerateFileContents(file_ID int) {
+	file.contents = "jhfvjhdfjhfjjhjhdfvjvcvfjh";
+}
+func (file HermesFile) GenerateFileChecksum() string {
+	
+	file_contents := []byte(file.contents)
+	hash := sha1.Sum(file_contents)
+	checksum := fmt.Sprintf("%s%x","_", hash)
+	return checksum;
+}
+func generate_file (id int) HermesFile {
```

Comment:
This one should be exported (a.k.a made public).
```suggestion
func GenerateHermesFile (id int) HermesFile {
```